.github/workflows/cs-lint.yml

```
@@ -69,7 +69,7 @@ jobs:
 
       # Check the code-style consistency of the PHP files.
       - name: Check PHP code style
-        run: vendor/bin/phpcs --report-full --report-checkstyle=./phpcs-report.xml
+        run: vendor/bin/phpcs --report-full --report-checkstyle=./phpcs-report.xml --severity=1
```

Any reason we shouldn't go ahead and change this to:
```suggestion
        run: composer cs -- --report-full --report-checkstyle=./phpcs-report.xml
```
...?

---

As long as it still works, I have no objections.